src/install/browserPaths.ts

```diff
@@ -77,9 +77,30 @@ export function executablePath(browserPath: string, browser: BrowserDescriptor):
   return tokens ? path.join(browserPath, ...tokens) : undefined;
 }
 
+const defaultBrowsersPath = ((): string | undefined => {
+  const env = getFromENV('PLAYWRIGHT_BROWSERS_PATH');
+  if (env === '0')
+    return undefined;
+
+  let result: string;
+  if (env)
+    result = env;
+  else if (process.platform === 'linux')
+    result = '~/.cache/playwright';
+  else if (process.platform === 'darwin')
+    result = '~/Library/Caches/playwright';
+  else if (process.platform === 'win32')
+    result = '~/AppData/Local/playwright';
+  else
+    throw new Error('Unsupported platform: ' + process.platform);
+
+  if (result.startsWith('~'))
```

Comment:
IIUC the expansion is done by certain shells, but not all. I wouldn't expand this on our side and leave this to the shell.

Reply:
Done